src/service.js

```diff
@@ -127,6 +125,9 @@ export function installServiceInEmbedScope(embedWin, id, service) {
       id,
       () => service
     );
+  } else {
+    registerServiceInternal(embedWin, embedWin, id, () => service);
+    getServiceInternal(embedWin, id); // Force service to build.
```

Comment:
Do we need to force build when the experiment is enabled above?

Reply:
I actually don't believe we need to. I mean the service is already built - we provide the instance. It's really just the structure initialization, but it will be correct anyway.